server/@types/journeys/index.d.ts

```diff
@@ -17,4 +17,21 @@ declare namespace journeys {
     isKnown: boolean
     dateOfBirth?: Date
   }
+
+  export interface ManageContactsJourney {
+    id: string
+    lastTouched: Date
+    search?: {
+      searchTerm?: string
+    }
+    prisoner?: {
+      firstName?: string
+      lastName?: string
+      prisonerNumber?: string
+      dateOfBirth?: string
+      prisonId?: string
+      prisonName?: string
+    }
```

Comment:
This is in anticipation of the prisoner-in-context for resulting forms in the journey.